spec/lib/cluster/config_checks/storage_spec.rb

```diff
@@ -0,0 +1,79 @@
+describe Cluster::ConfigChecks::Storage do
```

Comment:
There is tons of duplication in this spec compared to spec for Cluster::ConfigChecks::Database. It would not be super easy to refactor so I'm going to let this slide in the interest of moving forward.